src/editor.css

```diff
@@ -135,6 +135,7 @@
 .file-list-item .file-list-item-clipped {
   overflow: hidden;
   text-overflow: ellipsis;
+  padding-left: 30px;
```

Comment:
Let's set this property once on the rule that applies to both `clipped` and `unclipped`.

Reply:
would applying the property like this be alright? 

.file-list-item > div:not(.delete-file-button){
  padding-left: 30px;
}

Reply:
Yeah! You can also do a compound selector for explicitness:

```css
.file-list-item .file-list-item-clipped, 
.file-list-item .file-list-item-unclipped {
  padding-left: 30px;
}
```
If you go this route, make sure the rule is set after other padding rules for `file-list-item .file-list-item-clipped/unclipped`.